src/pages/notFound.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
There's some slightly odd behaviour for normal non-existent routes - by which I mean things like 
`https://mrc-ide.github.io/arbomap/malaria`. This redirects to `https://mrc-ide.github.io/arbomap/dengue/may24/arbomap/malaria` and we see:
```Sorry, this page does not exist!

Unknown indicator: arbomap. Unknown country ISO: malaria.
```

Reply:
This is tolerable but I thought I'd mention it